### ISSUE TITLE
Email Body is empty

### ISSUE THREAD

**Issue by polkonic** [Chocobozzz/PeerTube#2812]
Fresh installed, everything is working good except email body is missing. Apparently Postmarkapp cannot send empty email body. log shows email id, address, and subject line. However email body is empty. 
No email body.
am i missing something ?

{"message":{"accepted":["######@live.com"],"rejected":[],"envelopeTime":51,"messageTime":74,"messageSize":322,"response":"250 2.0.0 Ok: queued as 564BE405CAA","envelope":{"from":"#####@v.uyghur.life","to":["######@live.com"]},"messageId":"<51b08d4d-4656-e245-990d-#####@v.uyghur.life>","originalMessage":{"to":"######@live.com","from":"\"v.uyghur.life\" <######@v.uyghur.life>","subject":"[UyghurTube] Your video import https://www.youtube.com/watch?v=4nM9FBiUGeo is complete","attachments":[]}},"level":"info","label":"v.uyghur.life:443","timestamp":"2020-05-31T03:51:58.453Z"}

Subject line should be in the email body.

i am using latest 2.2.0-rc.1 with SMTP sender.

Peertube instance: https://v.uyghur.life/

**Comment by JohnXLivingston** [comment]
Seems related to https://github.com/Chocobozzz/PeerTube/issues/2794

**Comment by Findus23** [comment]
The same happens for me since updating to Peertube 2.2.0-rc.1.

This is the complete (redacted) E-Mail I receive as a notification. As one can see the whole Body is missing.
```http
Return-Path: <peertube@mypeertube.example>
Delivered-To: my@email.example
Received: [...]
DKIM-Signature: [...]
Content-Type: text/plain
From: "video.mypeertube.example" <peertube@mypeertube.example>
To: my@email.example
Subject: [PeerTube] New instance follower
Message-ID: <4108d4a7-7d1d-ba83-53cc-f8176783dea3@mypeertube.example>
Date: Sun, 31 May 2020 11:14:37 +0000
MIME-Version: 1.0
X-Rspamd-Queue-Id: B63B24081B
X-Rspamd-Score: -2.91 / 15.00 / 200.00
 ```

**Comment by rigelk** [comment]
@Findus23 @polkonic do you have logs that might be related?

**Comment by Findus23** [comment]
@rigelk 

The complete log corresponding to the above mail is:
```
info[31.5.2020, 23:15:40] Processing email in job 268.

info[31.5.2020, 23:15:40] Connection established to 127.0.0.1:25

{
  "component": "smtp-connection",
  "sid": "AfGQsPLVfEs",
  "tnx": "network",
  "localAddress": "127.0.0.1",
  "localPort": 53798,
  "remoteAddress": "127.0.0.1",
  "remotePort": 25
}

info[31.5.2020, 23:15:40] Sending message <26bf606a-67c5-faf5-1d28-49c1e46d80b2@mypeertube.example> to <my@email.example>

{
  "component": "smtp-transport",
  "tnx": "send",
  "messageId": "<26bf606a-67c5-faf5-1d28-49c1e46d80b2@mypeertube.example>"
}

info[31.5.2020, 23:15:40] <246 bytes encoded mime message (source size 243 bytes)>

{
  "component": "smtp-connection",
  "sid": "AfGQsPLVfEs",
  "tnx": "message",
  "inByteCount": 243,
  "outByteCount": 246
}

info[31.5.2020, 23:15:40] [object Object]

info[31.5.2020, 23:15:41] Connection closed

{
  "component": "smtp-connection",
  "sid": "AfGQsPLVfEs",
  "tnx": "network"
}
```

And checking the content of job 268 gives (redacted):
```json
{
  "to": [
    "my@email.example"
  ],
  "subject": "New instance follower",
  "text": "Your instance has a new follower: https://thenewfollower.example",
  "locals": {
    "title": "New instance follower",
    "action": {
      "text": "Review followers",
      "url": "https://mypeertube.example/admin/follows/followers-list"
    }
  }
}
```

**Comment by rigelk** [comment]
@Findus23 thanks for your help in documenting the issue! In the meantime, I could reproduce the bug on live servers, which happens for every kind of mail sent. It is however not at all reproducible in my dev setup or in the test suite, which makes it strange. More info on that tomorrow.

**Comment by JohnXLivingston** [comment]
Could it be a translation issue? As for dev setup, there is only english translations. Perhaps some translation files are missing or incomplete in the production build?

**Comment by rigelk** [comment]
@Findus23 @polkonic could you confirm you are using sendmail?

**Comment by rigelk** [comment]
> Could it be a translation issue?

No, since the internationalization of emails is not in place.

**Comment by polkonic** [comment]
> @Findus23 @polkonic could you confirm you are using sendmail?

No, SMTP,

config file:
`smtp:
  # smtp or sendmail
  transport: smtp
  # Path to sendmail command. Required if you use sendmail transport
  sendmail: null
  hostname: 'smtp.postmarkapp.com'
  port: 587
  username: '###############################'
  password: '###############################'
  tls: false
  disable_starttls: false
  ca_file: null # Used for self signed certificates
  from_address: 'admin@v.uyghur.life'

email:
  body:
    signature: "UyghurTube"
  subject:
    prefix: "[UyghurTube]"`

**Comment by McFlat** [comment]
I'm having the same issue. Emails are sent with empty body, SMTP mailgun, worked fine in 2.1.1

**Comment by rigelk** [comment]
This should now be fixed by 03fc19281b476f29b315083f373ca1a1fd4487f9, that will be included in 2.2.
